### PR TITLE
feat: implement userId for fromClaim directive

### PR DESCRIPTION
**Pull request by jensneuse**
Adds `@fromClaim(name: USERID)` to be able to inject the sub (mapped to userId) claim into Operations.

### CODE REVIEW

File: packages/nextjs-testapp/.wundergraph/wundergraph.config.ts [hunk 1]
```diff
@@ -93,11 +88,16 @@ const graphQLAPI = introspect.graphql({
         .addClientRequestHeader("Authorization","Authorization")
 });*/
 
+const mgoSpaceX = introspect.mongodb({
+	apiNamespace: 'mgospacex',
+	databaseURL: 'mongodb+srv://public:spacex@spacex-gcp-gpg0u.gcp.mongodb.net/spacex-api',
```
Comment (jensneuse) [inline packages/nextjs-testapp/.wundergraph/wundergraph.config.ts:93]: @StarpTech spaceX API fixed 🕺

Comment (StarpTech) [reply packages/nextjs-testapp/.wundergraph/wundergraph.config.ts:93]: :dancers:

**Review by StarpTech — CHANGES_REQUESTED** [review]
Please add the new claim options to the docs.

**Review by StarpTech — APPROVED** [review]
LGTM

**Pull request merged**